### DISCUSSION TITLE
Sending messages

### DISCUSSION THREAD

**Discussion by martinprejsa** [DislikesSchool/EduPage2#91] [Q&A] [answered]
Is sending messages a feature you plan to add? If yes, in what timeframe will it be available?

**Comment by vyPal** [comment, marked as answer]
Hello, yes sending messages is planned, I am currently working on a rewrite of the backend server, so that it is easier to implement such features. As for a timeframe, I don't really know yet. The rewrite isn't even close to finished. I might even cancel the rewrite, considering how long it's taking. I will post an update here when I start working on sending messages

**Reply by martinprejsa** [reply]
I'm up for help, i have experience with reverse engineering Edupage. I wrote you an email about it already.
